examples/barchart.html

```diff
@@ -71,6 +76,29 @@ <h1 class="my-5">Bar chart with filters</h1>
                 xAxisId: "age",
                 disaggregateById: "sex",
                 selectedFilterOptions: {}
+            },
+            formatFunction: (value, indicator) => {
+                const format = indicator.format;
+                const scale = indicator.scale;
+                const accuracy = indicator.accuracy;
```

Comment:
Could destructure here to reduce this to one line: `const { format, scale, accuracy } = indicator;`